ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -1,18 +1,23 @@
 import QtQuick 2.13
 
 import utils 1.0
+import StatusQ.Core.Utils 0.1 as StatusQUtils
 
 QtObject {
     id: root
 
     property var contactsStore
+
     property bool openCreateChat: false
-    property bool hideInput: false
-    property string createChatInitMessage
-    property var chatTextInput
+    property string createChatInitMessage: ""
+    property var createChatFileUrls: []
+    property bool createChatStartSendTransactionProcess: false
+    property bool createChatStartReceiveTransactionProcess: false
```

Comment:
I see this being set during send transaction but where is it used? I maybe be missing it

Reply:
@Khushboo-dev-cpp please check function `checkForCreateChatOptions` from the `ChatColumnView` component. There, depends on the certain case, we are using mentioned props.

The reason why we need them lies in the request of that feature (what I realized from the design figma files, but maybe we should check that). 

<table>
    <thead>
        <tr>
            <th rowspan=3>text message</th>
            <th rowspan=3>text + emoji message</th>
            <th rowspan=3>gif</th> 
            <th rowspan=3>sticker</th> 
            <th rowspan=3>receive or send transaction</th> 
            <th rowspan=3>send image</th> 
            <th colspan=2>Taken Action (pressing "Enter" or clicking "Confirm" button)</th>
        </tr>
        <tr>
            <th colspan=2>Selected contacts <br/>(if there is no sel contacts message input area is not displayed and "Confirm" button is disabled)</th>
        </tr>
        <tr>
            <th>one</th> 
            <th>more than one</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>X</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>Pressing "Enter" or "Confirm" creates `1:1` chat with "text message" as first message</td>
            <td>Pressing "Enter" or "Confirm" creates `group chat` with "text message" as first message</td>
        </tr>
        <tr>
            <td>/</td>
            <td>X</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>Pressing "Enter" or "Confirm" creates `1:1` chat with "text message + emoji" as first message</td>
            <td>Pressing "Enter" or "Confirm" creates `group chat` with "text message + emoji" as first message</td>
        </tr>
        <tr>
            <td>/</td>
            <td>/</td>
            <td>X</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>Click the gif automatically creates `1:1` chat with that gif as first message</td>
            <td>Click the gif automatically creates `group chat` with that gif as first message</td>
        </tr>
        <tr>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>X</td>
            <td>/</td>
            <td>/</td>
            <td>Click the sticker automatically creates `1:1` chat with that sticker as first message</td>
            <td>Click the sticker automatically creates `group chat` with that sticker as first message</td>
        </tr>
        <tr>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>X</td>
            <td>/</td>
            <td>Click the transaction button creates `1:1` chat and opens popup for sending/receiving transaction message</td>
            <td>Not available</td>
        </tr>
        <tr>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>/</td>
            <td>X</td>
            <td>Adds image to the chat input area which may be sent pressing "Enter" or "Confirm" button along with additionally added text and creates `1:1` chat and that image and text as first message</td>
            <td>Adds image to the chat input area which may be sent pressing "Enter" or "Confirm" button along with additionally added text and creates `1:1` chat and that image and text as first message</td>
        </tr>
    </tbody>
</table>